moto/appsync/models.py

```diff
@@ -321,13 +323,14 @@ def to_json(self) -> Dict[str, Any]:
             "userPoolConfig": self.user_pool_config,
             "xrayEnabled": self.xray_enabled,
             "visibility": self.visibility,
+            "tags": self.backend.list_tags_for_resource(self.arn),
         }
 
 
 class AppSyncBackend(BaseBackend):
     """Implementation of AppSync APIs."""
 
-    def __init__(self, region_name: str, account_id: str):
+    def __init__(self, region_name: str, account_id: str) -> None:
```

Comment:
Python typing tries to be smart here, which makes it confusing IMO. Because the return type of an `__init__` is always `None`, it doesn't need to be specified, **unless** there are no arguments.

So these examples are all valid:
`def __init__(self) -> None` --> No arguments, needs a return
`def __init__(self, arg: str)` --> All arguments typed, no return necessary
`def __init__(self, arg: str) -> None` --> All arguments typed, no return necessary, but also not wrong

Anyway, not a big deal either way - just an FYI :slightly_smiling_face: